codespell_lib/data/dictionary.txt

```diff
@@ -20645,11 +20711,15 @@ somethign->something
 somethimes->sometimes
 somethimg->something
 somethiong->something
+sometiem->sometimes
```

Comment:
Should be `sometime`

Reply:
In the location where I saw this typo, `sometimes` was clearly the right correction. Also, I feel like in code and reading some example sentences for `sometime`, the `sometimes` correction is mostly going to be the correct one. I will add sometime as a secondary correction though.

https://codesearch.debian.net/search?q=%5Cbsometiem%5Cb&literal=0
https://www.merriam-webster.com/dictionary/sometime